lib/datadog/tracing/contrib/redis/quantize.rb

```diff
@@ -29,7 +31,7 @@ module Quantize
           module_function
 
           def format_arg(arg)
-            str = arg.is_a?(Symbol) ? arg.to_s.upcase : arg.to_s
+            str = arg.to_s
             str = Core::Utils.utf8_encode(str, binary: true, placeholder: PLACEHOLDER)
```

Comment:
```suggestion
            str = Core::Utils.utf8_encode(str, binary: true, placeholder: PLACEHOLDER)
```

`Core::Utils.utf8_encode` will call `str.to_s` for you.